test/test_torch.py

```diff
@@ -7608,6 +7608,15 @@ def test_numpy_array_interface(self):
             for i in range(len(x)):
                 self.assertEqual(geq2_x[i], geq2_array[i])
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    def test_multiplication_numpy_scalar(self):
+        for np_dtype in [np.float32, np.float64, np.int32, np.int64]:
+            np_sc = np_dtype(2.0)
+            t = torch.ones(2, requires_grad=True)
+            r1 = t * np_sc
```

Comment:
Should `r1` all be `double`s (`t` is `double` because of default dtype is set as double in test)? Can you add an assert on the dtype?